libs/feature-workspaces/src/lib/workspaces/workspaces.component.scss

```diff
@@ -11,6 +11,102 @@ mat-list-item {
   overflow: hidden;
 }
 
+mat-nav-list {
+  margin-bottom: 24px;
+}
+
+@media (max-width: 920px) {
```

Comment:
Would love to pull these media queries into variables like `$small-screen`, `$medium-screen`, etc. Again, can do as a general cleanup chore later.

Reply:
Done!